tensorflow-notebook/Dockerfile

```diff
@@ -4,10 +4,28 @@ FROM jupyter/scipy-notebook
 
 LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
+
+USER $NB_USER
+
 # Install Tensorflow
+RUN pip install influxdb
+RUN pip install pyreadline
 RUN conda install --quiet --yes \
-    'tensorflow=1.3*' \
-    'keras=2.0*' && \
+    'numpy' \
+    'matplotlib' \
+    'pandas' \
+    'scipy' \
+    'seaborn' \
+    'scikit-learn' \
+    'scikit-image' \
+    'tensorflow' \
+    'keras' \
```

Comment:
I think we can get away with just updating the version numbers here without adding everything else. If there happens to be a version of numpy that also needs to be revised in the parent image, we should change that there.